mayastor/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -234,7 +235,10 @@ impl Nexus {
         child.destroy().await.context(DestroyChild {
             name: self.name.clone(),
             child: uri,
-        })
+        })?;
```

Comment:
if we fail to destroy here we don't restart the rebuilds?

Reply:
Good spot - fixed